doc/akri_connector/connector-metadata-schema.json

```diff
@@ -355,88 +390,28 @@
               "pattern": "^[a-zA-Z]+\\.[a-zA-Z]+",
               "type": "string"
             },
-            "additionalConfigurationSchema": {
-              "$ref": "http://json-schema.org/draft-07/schema#"
-            },
-            "assetEventDestinations": {
-              "type": "object",
-              "description": "Information about the supported and default destinations for asset events",
-              "properties": {
-                "supportedDestinations": {
-                  "type": "array",
-                  "description": "The array of destinations that this asset's events supports.",
-                  "items": {
-                    "type": "string",
-                    "enum": [
-                      "Mqtt",
-                      "Storage"
-                    ]
-                  }
-                },
-                "defaultDestination": {
-                  "description": "The destination that this asset's events will go to by default unless specified otherwise in the asset or the event.",
-                  "oneOf": [
-                    {"$ref" : "#/definitions/mqttDestinationDefaults"},
-                    {"$ref" : "#/definitions/storageDestinationDefaults"}
-                  ]
-                }
+            "supportedAuthenticationTypes": {
+              "type": "array",
+              "items": {
+                "type": "string"
               },
-              "additionalProperties": false,
-              "required": ["supportedDestinations"]
+              "description": "The set of authentication types that this connector supports. For example, [\"usernamePassword\", \"anonymous\"]"
```

Comment:
We currently only have one of three authentication methods available, is there a way to validate we're using one of those?

Reply:
It's possible to codify the allowed values here to be something like ["usernamePassword", "anonymous", ...], but that would limit users to our understanding of authentication. I'd rather we keep this open-ended so that we support future authentication mechanisms